DEPENDENCIES.md

```diff
@@ -0,0 +1,9 @@
+### Dependencies
+
+As we must vendor some libraries in, due to how Blender dependencies work, here
+is a list of included libraries.
+
+* python-gdtf: https://github.com/jackdpage/python-gdtf/
+* sACN / E1.31 module: https://github.com/Hundemeier/sacn
+* python-mvr: https://github.com/vanous/python-mvr
+* ArtnetPacket: https://gist.github.com/alarrosa14/07bd1ee88a19204cbf22
```

Comment:
Given the depth of changes to this gist, maybe we might keep it out? What do you think?

Reply:
Yes. The ArtNet code is on multiple places on GitHub and the only reason i added it was attribution, but this code has a good attribution already inside. I will remove it.